### DISCUSSION TITLE
Auto-renew option for domains

### DISCUSSION THREAD

**Discussion by wei** [vercel/vercel#5065] [Ideas] [answered]
Currently domains are always set to auto-renew unless they are removed from account/project many days before it actually expires. This results in 2 problems: 
1. Having to schedule something to go in and remove the domain many days before expiration.
1. Unable to use the domain purchased with vercel to its expiration, 

Customer support says one way around it would be to delete the payment method attached to the account. 🤦 
I'm hoping there could be a feature where users can toggle auto-renew status of domains instead since we still want to pay for the enterprise user seat.

Use case: The currently product we are building offers customers a 1 year website with custom domain. Not having the ability to disable auto-renew forces us to use a third party registrar. 

Thank you!

**Comment by paulogdm** [comment, marked as answer]
Hello Wei,

Using Vercel as a domain reseller **is not recommended** unless you have an agreement with us where this is clearly outlined. Domain's policies for registrar activities are somewhat complicated, and therefore, you do need to be covered legally and technically if you want to act as a domain reseller.

We are working to provide more features associated with domains in general. However, it is not a high priority item on our list since it is considered a edge-case. Vercel excels at hosting and workflow solutions, and you may need to consider other providers for domain-specific activities
